src/monetization.js

```
@@ -169,6 +169,35 @@ export class Monetization extends EventEmitter {
         return data;
     }
 
+    /**
+     * Checks if the user has access to a set of products or features.
+     * @param {array} pids - which products/features to check
+     * @param {array} userId - id of currently logged in user
+     * @throws {SDKError} - If a network call fails in any way (this will happen if, say, the user
+     * is not logged in)
+     * @returns {Object|null} The data object returned from Schibsted account (or `null` if the user
+     * doesn't have access to any of the given products/features)
+     */
+    async hasAccess(pids, userId) {
+        if (!this._sessionService || !userId) {
+            return null;
```

Not sure if this should throw an exception instead? WDYT @marjuszkiewicz ?